docs/showcase/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,26 @@
+
+
+include::_partials/attributes.adoc[]
+
+:toc:
+
+= Using the Showcase App
+
+include::service-setup.adoc[leveloffset=1,tags={include-tags}]
+
+
+// tag::excludeDownstream[]
```

Comment:
WOW. so if that is excluded downstream let's remove it then. 
Information there is kinda common problem and should be documented in stackoverflow rather than in our docs.